lib/src/sdk.ts

```diff
@@ -436,6 +451,127 @@ export class ExchangeSDK {
     return tx;
   }
 
+  /**
+   * Ask user to validate a fund transaction.
+   * @param {FundInfo} info - Information necessary to create a fund transaction.
+   * @return {Promise<string | void>} Promise of the hash of the sent transaction.
+   * @throws {ExchangeError}
+   */
+  async fund(info: FundInfo): Promise<string | void> {
+    this.flowType = "generic";
+    this.logger.log("*** Start Fund ***");
+
+    const {
+      fromAccountId,
+      fromAmount,
+      feeStrategy = FeeStrategyEnum.MEDIUM,
+      customFeeConfig = {},
+      orderId,
+      type = ExchangeType.CARD,
+    } = info;
+
+    const { account, currency } =
+      await this.walletAPIDecorator.retrieveUserAccount(
+        fromAccountId,
+        this.flowType
+      );
+
+    // Check enough funds
+    const initialAtomicAmount = this.convertToAtomicUnit(fromAmount, currency);
+    this.canSpendAmount(account, initialAtomicAmount);
+
+    // Step 1: Ask for deviceTransactionId
+    const deviceTransactionId = await this.exchangeModule
+      //TODO: pass in provider after updating startFund in LL
+      .startFund
+      // {provider: this.providerId}
```

Comment:
Do not forget add `fromAccountId` it will help us to install the network app if needed.
```suggestion
      // {provider: this.providerId, fromAccountId}
```

I think this value is missing too in sellStart, can you add it here or add a comment? I do not want to forget it. line 360